app/timetables/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
-# Register your models here.
+from app.timetables.models import Timetable
+
+admin.site.register(Timetable)
```

Comment:
@andela-joyebanji please add an extra line before admin.site.register to follow our convention of two line spaces after imports.